akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -153,6 +153,8 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
       .mapMaterializedValue(_.map(new ServerBinding(_))(ec).toJava))
   }
 
+  def newServerAt(interface: String): ServerBuilder = ServerBuilder(interface, system)
```

Comment:
I guess a `newServerAt(interface: String, port: Int)` overload would make sense.

Reply:
wouldn't `onPort` be sufficient?

Reply:
(arguably we could have `newServer` bind to localhost and have `onAddress` ;) - I'd say this is fine though)

Reply:
It would be somewhat more consistent to only have one way of doing things.

My idea of why to add another overload having the port was that you will basically always use that one. The situation where you don't want to specify a port is almost exclusively from tests. So, maybe we should instead remove this original overload and only have one with interface and port, so you'd have to be explicit about these at least.